hack/make-rules/test-e2e-node.sh

```diff
@@ -141,8 +141,9 @@ if [ $remote = true ] ; then
   exit $?
 
 else
-  # Refresh sudo credentials if not running on GCE.
+  # Refresh sudo credentials for local run
```

Comment:
Disagree with this nit.  This is the else to the following `if` statement:

```bash
if [ $remote = true ] ; then
```

To say 'not GCE' is not wrong, but 'local run' is more correct, since it's for any local run.